src/main/scala/BlockDevice.scala

```diff
@@ -1,20 +1,29 @@
 package testchipip
 
 import chisel3._
+import chisel3.experimental.{IO}
```

Comment:
This isn't in the main package?(!)

Reply:
Normally `IO` can only be called within a `Module`. This needs to be imported so we can add IO in a `InModuleBody`

Reply:
Ah, yes, thanks.